docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -1287,7 +1271,7 @@ Subhourly Forecast Parameters
    Time interval in minutes between the forecast model output files. If ``SUB_HOURLY_POST`` is set to "TRUE", this needs to be set to a two-digit integer between "01" and "59". Note that if ``SUB_HOURLY_POST`` is set to "TRUE" but ``DT_SUB_HOURLY_POST_MNTS`` is set to "00", ``SUB_HOURLY_POST`` will get reset to "FALSE" in the experiment generation scripts (there will be an informational message in the log file to emphasize this). 
 
 ..
-   COMMENT: In valid_param_vals.sh only these values are listed: "1" "01" "2" "02" "3" "03" "4" "04" "5" "05" "6" "06" "10" "12" "15" "20" "30". 
+   COMMENT: In valid_param_vals.sh only these values are listed: "1" "01" "2" "02" "3" "03" "4" "04" "5" "05" "6" "06" "10" "12" "15" "20" "30", but it says in this description that any value 00-59 can be used. Which is is?
 
```

Comment:
I think it's what is listed in the valid_param_vals.sh file.  Right now it's limited to 1-6, 10, 12, 15, 20, and 30-minute intervals when running in sub-hourly mode.